arangod/RocksDBEngine/RocksDBChecksumEnv.cpp

```diff
@@ -0,0 +1,234 @@
+///
```

Comment:
First line of standard header is missing
```suggestion
////////////////////////////////////////////////////////////////////////////////
///
```